addons/website_slides/controllers/main.py

```diff
@@ -370,6 +373,7 @@ def channel(self, channel, category=None, tag=None, page=1, slide_type=None, unc
             'user': request.env.user,
             'pager': pager,
             'is_public_user': request.website.is_public_user(),
+            'is_slides_publisher': request.env.user.has_group('website.group_website_publisher'),
```

Comment:
I would rename to is_website_publisher

Reply:
@tde-banana-odoo Shouldn't we use our shiny "channel.can_publish" here ?
And leave the name like it is then.

Reply:
Indeed, I also think we can directly use the channel.can_publish in the template, as we have the channel in sudo(user). and remove this 'is_slides_publisher' key.